provisioning/miniprov/hare_mp/main.py

```diff
@@ -389,10 +401,27 @@ def start_mkfs(hostname: str, hare_config_dir: str):
                           kv_file=f'{hare_config_dir}/consul-kv.json')
     cmd = '/usr/libexec/cortx-motr/motr-mkfs'
     # start mkfs for confd, ios services
+    start = perf_counter()
+    cmd_list = []
     for svc in ('confd', 'ios'):
         svc_fids = generator.get_svc_fids(svc)
         for fid in svc_fids:
-            execute([cmd, fid, '--conf'])
+            # real command line
+            logging.info(f'fid = {str(fid)}\n')
```

Comment:
```suggestion
            logging.info('Running mkfs [fid = %s]', fid)
```

Reply:
BTW this logging should happen within `start_mkfs()` function. Because at this point the work is planned only. There is no guarantee that this work will be processed immediately.

Reply:
Addressed.